### DISCUSSION TITLE
Welcome to recursive-visualizations discussions!

### DISCUSSION THREAD

**Discussion by pamelafox** [pamelafox/recursive-visualizations#19] [Announcements]
## 👋 Welcome!

If you have questions about using the tool or just want to share a neat example of how you're using it, this is the place for you! We'd love to see your recursive visualizations.  😍

**Comment by rajveerbumrah343** [comment]
Hello Ma’am,

I recently came across your website, recursionvisualizer.com and I must say it’s truly impressive. I’m currently working on a similar project to enhance my resume, and your site has been a great source of inspiration.

However, I’m having some difficulty understanding the WebAssembly (WASM) files used in your project. Could you kindly guide me on how to view or understand those files?

Thank you so much for your time and consideration.

Warm regards,
rajveer_ramgarhia343(instaid)

**Reply by pamelafox** [reply]
The website uses Pyodide (WASM version of Python) to run Python in the browser. You can learn more about Pyodide in their documentation:
https://pyodide.org/

I did not personally write any WASM myself.
